atomicsabi64/atomicsabi64.rst

```diff
@@ -1061,6 +1062,96 @@ compare-exchange. The result is returned in ``X0`` and ``X1``.
   +-------------------------------------+---------------+--------------------------------------+
 
 
+32-bit floating-point types
+---------------------------
+
+In what follows, register ``X1`` contains the location ``loc`` and ``S2``
+contains ``val``. The result is returned in ``S0``.
+
+.. table::
+
+  +-----------------------------------------------------+--------------------------------------+
+  | Atomic Operation                                    | AArch64                              |
+  +=====================================+===============+======================================+
+  | ``fetch_add(loc,val,relaxed)``      | ``Armv8-A``   | .. code-block:: none                 |
+  |                                     |               |                                      |
+  |                                     |               |    loop:                             |
+  |                                     |               |      LDXR   W0, [X1]                 |
+  |                                     |               |      FMOV   S1, W0                   |
+  |                                     |               |      FADD   S2, S2, S1               |
+  |                                     |               |      FMOV   W0, S2                   |
+  |                                     |               |      STXR   W3, W0, [X1]             |
+  |                                     |               |      CBNZ   W3, loop                 |
+  |                                     +---------------+--------------------------------------+
+  |                                     | ``FEAT_LSFE`` | .. code-block:: none                 |
+  |                                     |               |                                      |
+  |                                     |               |    LDFADD S0, S2, [X1] *             |
```

Comment:
The '*' does not apply to FP operations (no zero register).

Reply:
Thanks, updated.